split.py

```diff
@@ -0,0 +1,31 @@
+from sklearn.model_selection import train_test_split
+import pickle as pkl
+import pandas as pd
+from pathlib import Path
+
+import config
+import utils
+
+# Create split_data directory if it doesn’t exist
+Path(config.SPLIT_DATA_DIRECTORY_PATH).mkdir(parents=True, exist_ok=True)
+
+# Load processed data
+with open(config.EMOTIFY_SPECTROGRAM_DUMP_PATH, "rb") as f:
```

Comment:
For information, I have added in my PR a function that does the loading (you just have to provide the path). I will change these lines in my PR if you merge yours before. Otherwise could you use this function here ?